hodl/exchanges/PoloniexHelperAPI.py

```diff
@@ -10,6 +10,9 @@
 
 from hodl.ConfRetriever import ConfRetriever
 
+
+import datetime
```

Comment:
This should be up near line 7 with the other standard library imports.

For reference as to why please see the PEP8 Style Guidelines here: https://www.python.org/dev/peps/pep-0008/?#imports


> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.
> 
_hint: its a standard library import_